src/main/java/study2/view/InputView.java

```diff
@@ -0,0 +1,32 @@
+package study2.view;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Scanner;
+
+public class InputView {
+
+	Scanner scanner = new Scanner(System.in);
+	private static final String PARTICIPATEMESSAGES = "참여할 사람 이름을 입력하세요. (이름은 쉼표(,)로 구분하세요)";
+	private static final String HEIGHTMESSAGES = "최대 사다리 높이는 몇 개인가요?";
+
+	public List<String> participateName() {
```

Comment:
메소드 이름이 조금 모호한 것 같습니다.

participate는 동사로 어떤 행위를 하는 메소드 같은 느낌인데, 실제로는 `List<String>`을 반환하고 있어요.

```suggestion
	public List<String> getParticipantNames() {
```

Reply:
넵 적용하겠습니다.!